CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -957,17 +979,42 @@ local function Create3DSettingsIcon(topBarInstance, panel, menubar)
 
 		BackgroundTransparency = 1,
 
-		Image = "rbxasset://textures/ui/Menu/hamburger3D.png"
+		Image = "rbxasset://textures/ui/VR/hamburger.png"
 	}
 
+	local menuState = false
```

Comment:
This variable name isn't very descriptive of what it does, if it is used to keep track of whether the menu is open or not we should call it something like `menuOpened` or `menuOpenState`